src/Evolution/Systems/NewtonianEuler/Sources/LaneEmdenGravitationalField.cpp

```diff
@@ -0,0 +1,33 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/NewtonianEuler/Sources/LaneEmdenGravitationalField.hpp"
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/NewtonianEuler/LaneEmdenStar.hpp"
+#include "Utilities/Gsl.hpp"
+
+// IWYU pragma: no_include <array>
+
+// IWYU pragma: no_forward_declare Tensor
```

Comment:
Have you run IWYU on these files? If no, please remove the pragmas

Reply:
These pragmas are needed